README.md

```diff
@@ -8,19 +8,50 @@
 
 # Installation 
 
-1. `npm install screepsmod-auth` in your server folder.
-2. Thats it!
+1. Either run `npm install screepsmod-auth` in your screeps private server folder or subscribe the Steam workshop  https://steamcommunity.com/sharedfiles/filedetails/?id=800390576 if your using the in-game private server.
 
-# Usage
+2. Modify mods.json and add a new line in your mods list pointing to this mod like so:
 
-## Steam Console method
-1. Open the steam screeps client
-2. In console, run `setPassword('YourDesiredPassword')`
-3. Now you should be able to login via API
+```
+  "mods": [
+    "node_modules\\screepsmod-auth\\index.js"
+  ],
+```
+3. Relaunch your screeps private server.
+3. Follow the 
+[CLI steps](#screeps-server-command-line-interface-cli-method) below to setup your username and password 
+
+# Setup
+
+## Screeps Server Command Line Interface (CLI) method
+1. Connect to your screeps private server once to create your account.  You don't need to set an email as that won't be used.  Just username and password is important.
+
+2. Run the server's command line interface (CLI) either in Steam, or if your running a standalone server as described here https://github.com/screeps/screeps#command-line-interface-cli
+
+3. In server's CLI, run 
+
+```
+setPassword('yourUserName', 'yourPassword')
+```
+Note: that setPassword says to pass in the email.  Don't do this.  Instead give it your username.
```

Comment:
If an email is set during step 1, it will accept either email or username

Reply:
Just checked the code again, as I could have sworn I updated the README, trying setPassword without the correct params does actually say use username, and the code doesn't accept email for this one.

Reply:
Alright, I'll leave it as it is as I think your saying my readme edits are correct right?